pkgs/by-name/cu/cuberite/package.nix

```
@@ -0,0 +1,77 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  lua,
+  perl,
+  python3,
+  unstableGitUpdater,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "cuberite";
+  version = "0-unstable-2024-09-08";
+
+  src = fetchFromGitHub {
+    owner = "cuberite";
+    repo = "cuberite";
+    rev = "0325de7dacaf1e2feaea5eaab4791bc23d78f9e7";
+    fetchSubmodules = true; # TODO: Switch to nixpkgs-packaged dependencies
+    hash = "sha256-Gz7WUBcQcUnEs9LD3/Epi9RBZrERCVgLtIlDH1tiK4E=";
+  };
+
+  postPatch = ''
+    patchShebangs .
+  '';
+
+  nativeBuildInputs = [
+    cmake
+    python3
+  ];
+  buildInputs = [
+    lua
+    perl
+  ];
```

```suggestion
  nativeBuildInputs = [
    cmake
    python3
    perl
  ];
  buildInputs = [
    lua
  ];
```
I am not sure lua is required, the output certainly doesn't depend on it.